client/lib/importer/actions.js

```diff
@@ -9,7 +9,25 @@ const wpcom = require( 'lib/wp' ).undocumented();
 /**
  * Internal dependencies
  */
-import { actionTypes, appStates } from './constants';
+import {
+	IMPORTS_AUTHORS_SET_MAPPING,
+	IMPORTS_AUTHORS_START_MAPPING,
+	IMPORTS_FETCH,
+	IMPORTS_FETCH_FAILED,
+	IMPORTS_FETCH_COMPLETED,
+	IMPORTS_IMPORT_CANCEL,
+	IMPORTS_IMPORT_LOCK,
+	IMPORTS_IMPORT_RECEIVE,
+	IMPORTS_IMPORT_RESET,
+	IMPORTS_IMPORT_START,
+	IMPORTS_IMPORT_UNLOCK,
+	IMPORTS_START_IMPORTING,
```

Comment:
Not sure. The naming for starting the import process has never been great because we have multiple start signals. This particular one indicates that after uploading and mapping authors, the actual import process should begin. Any suggestions? It's hard to differentiate from `IMPORTS_IMPORT_START`